src/renderer/actions.ts

```diff
@@ -153,8 +154,11 @@ export function login(): ThunkAction {
                                           password);
         } catch (error) {
             console.info('error: ', error);
-            //showToast({type: 'error', text: 'Wrong username or password.'});
-            showToast({type: 'error', text: 'Wrong username or password.'});
+            if (error instanceof HttpError && (error.status === 401 || error.status === 403)) {
```

Comment:
I think it is an easier clue for support staff reporting unauthorized when the server returns status 401. 401s are not caused by a wrong username and password but by e.g. wrong client credentials.

```suggestion
            if (error instanceof HttpError && (error.status === 401)) {
                showToast({type: 'error', text: 'Access unauthorized.'});
            }
            else if(error instanceof HttpError && (error.status === 403)) {
```

Reply:
But 401 is what I get if I enter "normando" instead of "norman".